src/components/color_picker/color_picker.xml

```diff
@@ -20,9 +20,18 @@
             </div>
           </div>
         </div>
-        <div class="o-separator"/>
-        <div class="o-color-picker-section-name">
-          <span>Custom</span>
+      </div>
+      <div class="o-separator"/>
+      <div class="o-color-picker-section-name">
+        <span>Custom</span>
+      </div>
+      <div class="colors-grid">
```

Comment:
Be careful while rebasing, you duplicated this div. Couldn't be seen in the tests though since we did not have a snapshot of the html in the tests (which I add now)

Reply:
Okay, thankyou though